File: P1085/main.py
hour = [None] * 2
week = [hour] * 7
happiness =[0] * 7
index = 0
max = happiness[0]
happy = True

for i in range (7):
    week[i] = list(map(int, input().split()))

for i in range (7):
    happiness[i] = week[i][0] + week[i][1]

for i in range (7):
    if happiness[i] > max :
        max = happiness[i]
        index = i

if max > 8 :
    print(index + 1)
    happy == False
elif happy == True:
    print(0)

File: P1085/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

    vector<vector<int>>  week(7, vector<int>(2));
    vector<int> happiness(7);
    int index = 0;
    int max = happiness.at(0);
    bool happy = true;

    for (int i = 0; i < 7; i++){
        cin >> week.at(i).at(0) >> week.at(i).at(1);
    }

    for (int i = 0; i < 7 ; i++){
        happiness.at(i) = week.at(i).at(0) + week.at(i).at(1);
    }

    for (int i = 0; i < 7; i++){
        if (happiness.at(i) > max){
            max = happiness.at(i);
            index = i;
        }
    }

    if ( max > 8) {
        cout << index + 1;
        happy = false;
    } else if (happy == true)
    {
       cout << 0;
    }
    return 0;
}